src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -1500,7 +1511,7 @@ LogicalResult ONNXConvOp::inferShapes() {
 LogicalResult ONNXConvTransposeOp::inferShapes() {
   // Generic shape for data input X, weight tensor W, and optional bias B
   // X: (N x C x D1 x D2 ... x Dn)
-  // W: (M x C/group x k1 x k2 x ... x kn)
+  // W: (C x M/group x k1 x k2 x ... x kn)
```

Comment:
This follows the pytorch/onnx spec more closely - looks like it was a copy-paste error before.

Reply:
thanks, it was an error indeed